editor/animation_track_editor.cpp

```diff
@@ -2854,7 +2854,7 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 				bool selected = _try_select_at_ui_pos(pos, mb->is_command_or_control_pressed() || mb->is_shift_pressed(), false);
 
 				menu->clear();
-				menu->add_icon_item(get_editor_theme_icon(SNAME("Key")), TTR("Insert Key..."), MENU_KEY_INSERT);
+				menu->add_icon_item(get_editor_theme_icon(SNAME("Key")), TTR("Insert Key"), MENU_KEY_INSERT);
```

Comment:
I'd say this should be dependent on the track type, some tracks *do* spawn a menu, like method tracks

Reply:
This may be the same as "Save", which conditionally pops up a dialog. It does not use ellipses conventionally.

But anyway, adjusting the text based on whether user interaction is needed is always an option :P

Reply:
I'd say it's different as it's not conditional on if the file is already saved, but on the track type, but good point